_spinnaker/armory_halyard.md

```diff
@@ -49,8 +49,9 @@ hal armory [subcommands]
 ```
 
 #### Subcommands
- * `diagnostics`: configure diagnostics reporting
- * `dinghy`: configure Dinghy pipelines as code
+ * `secrets`: Configure secrets management
```

Comment:
Alphabetical order?

Reply:
everything in the docs and the cli help appears to be in alphabetical order, so i was keeping it consistent